<div fxLayout="column" fxLayoutGap="10px" fxFill>
  <div fxFlex="none">
    <h1 class="page-header">Document</h1>
  </div>

  <div fxFlex class="not-float">
    <mat-card appMaterialElevation defaultElevation="4" raisedElevation="4">
      <mat-card-header>
        <mat-card-title>Introduction</mat-card-title>
      </mat-card-header>
      <mat-card-content>
        <p>
          GSCA (Gene Set Cancer Analysis) is an updated version of
          <a href="http://bioinfo.life.hust.edu.cn/web/GSCALite/" target="_blank">GSCALite</a>, which is a web-based analysis platform for
          gene set cancer analysis relate to mRNA expression, mutation, immune infiltrates and drug resistance. The platform mainly based on
          TCGA and GDSC database. <br />
          The gene set alteration on DNA or RNA are received more and more attentions in recent years, which may be important in cancer
          initiation, progress, diagnosis, prognosis and therapy. As the cancer genomics big data available, it is very useful and urgent to
          provide a platform for gene set analysis in cancer. <br />
          By using GSCA, experimental biologists can easily explore the large TCGA and drug response datasets, test hypotheses relate to
          genomics changes, and their association with clinical outcome and immune infiltration in a higher resolution.
        </p>
        <h3>Abstract Summary</h3>
        <img mat-card-image class="zm" [src]="assets + '/imgs/' + 'zm.png'" alt="Abstrat_img" />
      </mat-card-content>
    </mat-card>
    <br />

    <mat-card appMaterialElevation defaultElevation="4" raisedElevation="4">
      <mat-card-header>
        <mat-card-title>Quick start</mat-card-title>
      </mat-card-header>
      <mat-card-content>
        <ol>
          <li>
            <p>
              Click "Example" button to get example inputs. Or enter the official symbols of gene set with more than 2 genes and less than
              50 in the “Input your interested gene set”.
            </p>
          </li>
          <li>
            <p>Select at least one cancer type in the "Select cancer types".</p>
          </li>
          <li>
            <p>Select at least one analysis.</p>
          </li>
          <li>
            <p>Click "Submit" button to get results of interest gene set.</p>
          </li>
        </ol>
        <img mat-card-image class="zm" [src]="assets + '/imgs/' + 'zm.png'" alt="searchbox" />
      </mat-card-content>
    </mat-card>
    <br />
    <mat-card appMaterialElevation defaultElevation="4" raisedElevation="4">
      <mat-card-header>
        <mat-card-title>Updates of GSCA compared with GSCALite</mat-card-title>
      </mat-card-header>
      <mat-card-content>
        <p>GSCA retained all the functions in the GSCALite. Besides, in this version, more datasets and functions were added:</p>
        <ol>
          <li>
            <p>
              Click "Example" button to get example inputs. Or enter the official symbols of gene set with more than 2 genes and less than
              50 in the “Input your interested gene set”.
            </p>
          </li>
          <li>
            <p>Select at least one cancer type in the "Select cancer types".</p>
          </li>
          <li>
            <p>Select at least one analysis.</p>
          </li>
          <li>
            <p>Click "Submit" button to get results of interest gene set.</p>
          </li>
        </ol>
        <img mat-card-image class="zm" [src]="assets + '/imgs/' + 'zm.png'" alt="searchbox" />
      </mat-card-content>
    </mat-card>
    <br />

    <mat-card appMaterialElevation defaultElevation="4" raisedElevation="4">
      <mat-card-header>
        <mat-card-title>Quick start</mat-card-title>
      </mat-card-header>
      <mat-card-content>
        <ol>
          <li>
            <h3>Gene set estimation</h3>
            <p>
              This function allows users to estimate the overall estimation of the gene set in the aspect of mRNA expression, SNV and CNV
              levels. This make it possible to carry out calculation in the gene set level instead of one gene. For instance, GSVA analysis
              was performed to estimate the overall expression level of gene set for each TCGA sample.
            </p>
          </li>
          <li>
            <h3>Association with immune infiltrates</h3>
            <p>
              This function allows users to estimate the association of genomic features (RNA sequencing expression, SNV, CNV and
              methylation) with 25 immune cells infiltration, evaluated by ImmuCellAI.
            </p>
          </li>
          <li>
            <h3>Detailed single gene result</h3>
            <p>
              This function provides results of single gene for batch calculation, which avoid fussy input options when the users have
              plenty of candidate genes.
            </p>
          </li>
          <li>
            <h3>Cancer TNM Stage</h3>
            <p>
              This function allows the comparison of gene expression between TNM stages. A trend analysis was utilized to get the trend of
              expression as the disease progresses.
            </p>
          </li>
        </ol>
        <img mat-card-image class="zm" [src]="assets + '/imgs/' + 'zm.png'" alt="searchbox" />
      </mat-card-content>
    </mat-card>
    <br />

    <mat-card appMaterialElevation defaultElevation="4" raisedElevation="4">
      <mat-card-header>
        <mat-card-title>Data collection and process</mat-card-title>
      </mat-card-header>
      <mat-card-content>
        <ul>
          <li>
            <h3>TCGA data from NCI Genomic Data Commons</h3>
            <h4>mRNA sequencing data</h4>
            <ul>
              <li>
                Statistic
                <ul>
                  <li>33 cancer types</li>
                  <li>10,995 samples</li>
                </ul>
              </li>
              <li>
                Data process
                <ul>
                  <li>14 cancers have over ten paired tumor and normal samples.</li>
                  <li>Processed with RSEM normalized expression</li>
                </ul>
              </li>
            </ul>
            <h4>Clinical information</h4>
            <ul>
              <li>
                Survival
                <ul>
                  <li>xx cancer types</li>
                  <li>xx samples</li>
                  <li>overall survival(OS) and progression free survival(PFS)</li>
                </ul>
              </li>
              <li>
                Stage
                <ul>
                  <li>xx cancer types</li>
                  <li>xx samples</li>
                  <li>including stage I, II, III, IV</li>
                </ul>
              </li>
              <li>
                Subtype
                <ul>
                  <li>xx cancer types</li>
                  <li>xx samples</li>
                  <li>including molecular subtypes or clustered subtypes (numbered)</li>
                </ul>
              </li>
            </ul>
            <h4>SNV data</h4>
            <ul>
              <li>
                Statistic
                <ul>
                  <li>33 cancer types</li>
                  <li>8,663 samples</li>
                </ul>
              </li>
              <li>
                Data process
                <ul>
                  <li>
                    Mutant including 7 types of mutations: Missense_Mutation, Nonsense_Mutation, Frame_Shift_Ins, Splice_Site,
                    Frame_Shift_Del, In_Frame_Del, In_Frame_Ins.
                  </li>
                  <li>Wide type(WT): others.</li>
                </ul>
              </li>
            </ul>
            <h4>CNV data</h4>
            <ul>
              <li>
                Statistic
                <ul>
                  <li>33 cancer types</li>
                  <li>11,495 samples</li>
                </ul>
              </li>
              <li>
                Data process
                <ul>
                  <li>GISTICS2.0</li>
                  <li>Hete Amp: heterozygous amplification, CNV = 1</li>
                  <li>Hete Del: heterozygous deletion, CNV = -1</li>
                  <li>Homo Amp: homozygous amplification, CNV = 2</li>
                  <li>Homo Del: homozygous deletion, CNV = -2</li>
                </ul>
              </li>
            </ul>
            <h4>Methylation data</h4>
            <ul>
              <li>
                Statistic
                <ul>
                  <li>33 cancer types</li>
                  <li>10,129 samples</li>
                </ul>
              </li>
              <li>
                Data process
                <ul>
                  <li>Calculation based on methylation beta value</li>
                  <li>14 cancers have over ten paired tumor and normal samples</li>
                </ul>
              </li>
            </ul>
          </li>
          <li>
            <h3>Drug data</h3>
            <h4>Therapeutics Response Portal (CTRP)</h4>
            <ul>
              <li>481 small molecules IC50</li>
              <li>1001 cell lines</li>
              <li>18900 genes</li>
            </ul>
            <h4>Genomics of Drug Sensitivity in Cancer (GDSC)</h4>
            <ul>
              <li>265 small molecules IC50</li>
              <li>860 cell lines</li>
              <li>17419 genes</li>
            </ul>
          </li>
          <li>
            <h3>Immune infiltrates</h3>
            <ul>
              <li>
                Statistic
                <ul>
                  <li>TCGA 10,995 samples</li>
                  <li>33 cancer types</li>
                  <li>25 immune cells</li>
                </ul>
              </li>
              <li>
                Data process
                <ul>
                  <li>Based on mRNA expression data, ImmuCellAI was performed to get immune cells infiltrates</li>
                  <li>25 immune cells</li>
                </ul>
              </li>
            </ul>
          </li>
        </ul>
      </mat-card-content>
    </mat-card>
    <br />
  </div>

  <div fxFlex="none">
    <app-footer></app-footer>
  </div>
</div>
