<mat-card fxFlex appMaterialElevation defaultElevation="0" raisedElevation="0">
  <mat-expansion-panel hideToggle>
    <mat-expansion-panel-header>
      <mat-panel-title> Click here for the detailed description of methods and results </mat-panel-title>
      <mat-panel-description></mat-panel-description>
    </mat-expansion-panel-header>
    <ul>
      <p>Methylation & survival module provides survival analysis for high and low methylation groups.</p>
      <li>Methods</li>
      <p>
        Illumina HumanMethylation 450k level 3 data were downloaded from TCGA database. Generally, there are multiple methylation sites in
        the region of one gene. Therefore, there are multiple tags store the methylation level of each site. Before survival analysis, we
        performed correlation analysis to filter <span class="phighlight">the site most negatively correlate</span> with gene expression
        into this analysis.
      </p>
      <p>
        GSCA downloaded clinical data of tumor samples from 33 cancer types from TCGA and
        <a href="https://www.cell.com/cell/abstract/S0092-8674(18)30229-0" target="_blank">(Liu et al, Cell, 2018)</a>. Uncensored data was
        left out. Samples that have <span class="phighlight">competing risk</span> for death from cancer were filtered out (for DSS and DFI
        data).
      </p>
      <p>
        Methylation data and clinical survival data were merged by sample barcode, median methylation level was used to divide tumor samples
        into high and low methylation groups.
      </p>
      <p>
        <span class="phighlight">R package survival</span> was used to fit survival time and survival status within two groups.
        <span class="phighlight">Cox Proportional-Hazards</span> model was constructed to get the risk ratio (Hazard ratio) of high
        methylation group compared with low methylation group. A <span class="phighlight">Logrank test</span> was performed to test whether
        the survival difference between the groups is statistically significant.
      </p>
      <p>
        Note that the users may <span class="phighlight">could not get the survival results</span> in several cases, and there are some
        suggestions for survival analysis:
      </p>
      <ul>
        <li class="lilight">
          1. The OS data are available for all cancer types, while survival analysis for OS is not recommended for DLBC, PCPG, TGCT, and
          THYM
          <a href="https://www.cell.com/cell/abstract/S0092-8674(18)30229-0" target="_blank">(Liu et al, Cell, 2018)</a>.
        </li>
        <li class="lilight">
          2. The PFS data are not available for LAML. Survival analysis for PFS is not recommended for PCPG
          <a href="https://www.cell.com/cell/abstract/S0092-8674(18)30229-0" target="_blank">(Liu et al, Cell, 2018)</a>.
        </li>
        <li class="lilight">
          3. The DSS data are not available for LAML. Survival analysis for DSS is not recommended for DLBC, PCPG, PRAD, TGCT, THCA, and
          THYM<a href="https://www.cell.com/cell/abstract/S0092-8674(18)30229-0" target="_blank">(Liu et al, Cell, 2018)</a>.
        </li>
        <li class="lilight">
          4. The DFI data are not available for LAML, SKCM, THYM, and UVM. Survival analysis for DFI is not recommended for DLBC, GBM, KICH,
          MESO, PCPG, and READ<a href="https://www.cell.com/cell/abstract/S0092-8674(18)30229-0" target="_blank">(Liu et al, Cell, 2018)</a
          >.
        </li>
      </ul>

      <li>Interpretation</li>
      <p>GSCA provides two forms of results, including table and figure.</p>

      <ul>
        <li>The result table:</li>
        <p>
          Presents detailed information about Cancer type, Gene symbol, Hazard Ratio, Cox P value, Logrank P value, Higher risk of death.
          This table showing the result of OS.
        </p>
        <p>
          Higher risk of death indicates the group with higher risk of death. When the <span class="phighlight">Hazard ratio</span> of high
          methylation group compared with low methylation group is <span class="phighlight">greater than 1</span>, we think higher
          methylation group has higher risk of death, else lower methylation group has higher risk of death.
        </p>
        <p>Tag: the tag id of methylation site within the region of the specific gene.</p>

        <li>GSCA provides result figures including:</li>
        <ul>
          <li>Bubble plot:</li>
          <p>
            Presents the Hazard ratio and Cox p value through bubble color and size. The column is gene symbols and row is the selected
            cancer types. The bubble color from blue to red represents the hazard ratio from low to high, the bubble size is positively
            correlate with the Cox P value significance. Black outline border indicates Cox P value â‰¤ 0.05.
          </p>

          <li>Kaplan-Meier curves:</li>
          <p>
            By clicking buttons in table result, GSCA draws Kaplan-Meier curves between methylation groups of specific gene in specific
            cancer.
          </p>
        </ul>
      </ul>
    </ul>
  </mat-expansion-panel>
</mat-card>
