<mat-card fxFlex appMaterialElevation defaultElevation="0" raisedElevation="0">
  <mat-expansion-panel hideToggle>
    <mat-expansion-panel-header>
      <mat-panel-title>
        <mat-icon class="pred">info</mat-icon>Click here for the detailed description of methods and results.
      </mat-panel-title>
      <mat-panel-description></mat-panel-description>
    </mat-expansion-panel-header>
    <ul>
      <p>Methylation & survival module provides survival analysis for high and low methylation groups.</p>
      <li>Methods</li>
      <p>
        Illumina HumanMethylation 450k level 3 data were downloaded from TCGA database. Generally, there are multiple methylation sites in
        the region of one gene. Therefore, there are multiple tags store the methylation level of each site. Before survival analysis, we
        performed correlation analysis to filter <span class="phighlight">the site most negatively correlate</span> with gene expression
        into this analysis.
      </p>
      <p>
        GSCA downloaded clinical data of tumor samples from 33 cancer types from TCGA and
        <a href="https://www.cell.com/cell/abstract/S0092-8674(18)30229-0" target="_blank">(Liu et al., Cell, 2018)</a>. Uncensored data was
        left out. Samples that have <span class="phighlight">competing risk</span> for death from cancer were filtered out (for DSS and DFI
        data).
      </p>
      <p>
        Methylation data and clinical survival data were merged by sample barcode, median methylation level was used to divide tumor samples
        into high and low methylation groups.
      </p>
      <p>
        <span class="phighlight">R package survival</span> was used to fit survival time and survival status within two groups.
        <span class="phighlight">Cox Proportional-Hazards</span> model was constructed to get the risk ratio (Hazard ratio) of high
        methylation group compared with low methylation group. A <span class="phighlight">Logrank test</span> was performed to test whether
        the survival difference between the groups is statistically significant.
      </p>
      <p><span class="phighlight">There are some suggestions for survival analysis:</span></p>
      <ul>
        <li class="lilight">Data availability</li>
        <p>
          The OS data are available for all cancer types.<br />
          The PFS data are not available for LAML.<br />
          The DSS data are not available for LAML.<br />
          The DFI data are not available for LAML, SKCM, THYM, and UVM.
        </p>
        <li class="lilight">
          Attention for using the endpoints of OS, PFI, DFI, and DSS (<a
            href="https://www.cell.com/cell/abstract/S0092-8674(18)30229-0"
            target="_blank"
            >Liu et al., Cell, 2018</a
          >).
        </li>
        <p>
          ACC: number of events is small; <br />
          BRCA: need a longer follow-up for OS and DSS; <br />
          CHOL: sample size is too small for OS, DSS, DFI, and PFI; <br />
          DLBC: sample size and number of events are too small, need a longer follow-up; <br />
          GBM: number of disease-free cases is small;<br />
          KICH: number of events is too small, need a longer follow-up; <br />
          KIRC: number of events is small; <br />
          LGG: need a longer follow-up for OS and DSS; <br />
          LIHC: need a longer follow-up DSS; <br />
          MESO: sample size for DFI is small; <br />
          PCPG: need a longer follow-up for OS, DSS, DFI, and PFI; number of events is small; <br />
          PRAD: need a longer follow-up for OS and DSS;<br />
          READ: need a longer follow-up for OS, DSS, and DFI; number of events for DFI is too small; <br />
          TGCT: number of events is small for OS and DSS; need a longer follow-up; <br />
          THCA: number of events is small for OS and DSS; need a longer follow-up; <br />
          THYM: number of events is too small for OS and DSS; need a longer follow-up; <br />
          UCS: sample size is small.<br />
        </p>
      </ul>

      <li>Interpretation</li>
      <p>GSCA provides two forms of results, including table and figure.</p>

      <ul>
        <li>The result table:</li>
        <p>
          The table presents detailed information about Cancer type, Gene symbol, Hazard Ratio, Cox P value, Logrank P value, Higher risk of
          death. This table showing the result of OS.
        </p>
        <p>
          Higher risk of death indicates the group with higher risk of death. When the <span class="phighlight">Hazard ratio</span> of high
          methylation group compared with low methylation group is <span class="phighlight">greater than 1</span>, we think higher
          methylation group has higher risk of death, else lower methylation group has higher risk of death.
        </p>
        <p>Tag: the tag id of methylation site within the region of the specific gene.</p>

        <li>GSCA provides result figures, including:</li>
        <ul>
          <li>Bubble plot:</li>
          <p>
            Presents the Hazard ratio and Cox p value through bubble color and size. The column is gene symbols and row is the selected
            cancer types. The bubble color from blue to red represents the hazard ratio from low to high, the bubble size is positively
            correlate with the Cox P value significance. Black outline border indicates Cox P value â‰¤ 0.05.
          </p>

          <li>Kaplan-Meier curves:</li>
          <p>
            By clicking any one button in the table, GSCA would draw the Kaplan-Meier curves for the methylation groups of specific gene in
            specific cancer.
          </p>
        </ul>
      </ul>
    </ul>
  </mat-expansion-panel>
</mat-card>
